lib/matplotlib/backends/backend_pdf.py

```diff
@@ -348,11 +348,22 @@ class Stream:
     __slots__ = ('id', 'len', 'pdfFile', 'file', 'compressobj', 'extra', 'pos')
 
     def __init__(self, id, len, file, extra=None, png=None):
-        """id: object id of stream; len: an unused Reference object for the
-        length of the stream, or None (to use a memory buffer); file:
-        a PdfFile; extra: a dictionary of extra key-value pairs to
-        include in the stream header; png: if the data is already
-        png compressed, the decode parameters"""
+        """
+        Parameters
+        ----------
+
+        id : int
+            object id of the stream
```

Comment:
```suggestion
            Object id of the stream.
```

Generally, parameter descriptions should start with a capital letter and finish with a period, even if they are not complete sentences.

(Also in the following docs).